### DISCUSSION TITLE
Project Brainstorm

### DISCUSSION THREAD

**Discussion by hamiecod** [rovaay/.github#2] [Ideas]
Discussion for project ideas :D

**Comment by hamiecod** [comment]
We should make some projects for rovaay

**Comment by hamiecod** [comment]
I have a little idea, we can use the GitHub API to dynamically generate an image of this chat with cool effect applied on that image. Then we can add it to the GitHub README of the organization. I have seen a similar thing on somebody's GitHub profile but I do not remember the username as of now.

**Reply by hamiecod** [reply]
Mention: @robiot

**Reply by robiot** [reply]
Yeah

**Reply by hamiecod** [reply]
Should we build it?
